android/quest/src/test/java/org/smartregister/fhircore/quest/util/extensions/ConfigExtensionsTest.kt

```diff
@@ -282,6 +284,52 @@ class ConfigExtensionsTest : RobolectricTest() {
     Assert.assertEquals("geoWidgetId", slotBundle.captured.getString(NavigationArg.CONFIG_ID))
   }
 
+  @Test
+  fun testLaunchDiallerOnClick() {
+    val patientWithPhoneNumber = patient.copy()
+    patientWithPhoneNumber.apply {
+      addTelecom(
+        ContactPoint().apply { this.value = "0700000000" },
+      )
+    }
+
+    val computedValuesWithPhoneNumberMutable = resourceData.computedValuesMap.toMutableMap()
+    computedValuesWithPhoneNumberMutable["patientPhoneNumber"] =
+      patientWithPhoneNumber.telecom.first().value
+    val computedValuesWithPhoneNumber = computedValuesWithPhoneNumberMutable.toMap()
+
+    val resourceDataWithPhoneNumber =
+      ResourceData(
+        baseResourceId = patient.logicalId,
+        baseResourceType = ResourceType.Patient,
+        computedValuesMap = computedValuesWithPhoneNumber,
+      )
+
+    val clickAction =
+      ActionConfig(
+        id = "diallerId",
+        trigger = ActionTrigger.ON_CLICK,
+        workflow = ApplicationWorkflow.LAUNCH_DIALLER.name,
+        params =
+          listOf(
+            ActionParameter(
+              key = "patientPhoneNumber",
+              value = "@{patientPhoneNumber}",
+              paramType = ActionParameterType.PARAMDATA,
+            ),
+          ),
+      )
+
+    listOf(clickAction)
+      .handleClickEvent(
+        navController = navController,
+        resourceData = resourceDataWithPhoneNumber,
+      ) // make a clicking action
+
+    // make sure no errors thrown when the new activity is started. should return nothing
+    every { context.startActivity(any()) } returns Unit
+  }
```

Comment:
This test does not appear to be complete. There are no `verifications` or `assertions` implemented.